src/frontend/hooks/useProjectInvite.ts

```diff
@@ -0,0 +1,33 @@
+import {Invite} from '@mapeo/core/dist/invite-api';
+import {useAcceptInvite, useInvites, useRejectInvite} from './server/invites';
+import {useApi} from '../contexts/ApiContext';
+import {useCallback} from 'react';
+
+export function useProjectInvite() {
+  const invites = useInvites().data;
+  // this will eventually sort invite by date
+  const invite = invites[0];
+  const acceptMutation = useAcceptInvite(invite?.inviteId);
+  const rejectMutation = useRejectInvite(invite?.inviteId);
+  const mapeoApi = useApi();
+
+  const resetState = () => {
+    acceptMutation.reset();
+    rejectMutation.reset();
+  };
```

Comment:
Becaues the hook is inside a modal, and the modal is at the top level, the modal and the hook never get unmounted (a modal just gets hidden, it doesn't unmount when closed). Because it never unmounts the state of the modal is not automatically reset. So, when we close the modal we want to make sure that we reset the state so the user does not see the last state